frontend/src/components/design-system/containers/PillComponent.vue

```diff
@@ -1,17 +1,43 @@
 <script setup lang="ts">
+import { computed } from 'vue';
 import LabelText from '@/components/design-system/typography/LabelText.vue';
 
 export interface Props {
   active?: boolean;
 }
 
-withDefaults(defineProps<Props>(), {
+const classes = computed(() => ({
+  'pill--active': props.active,
+}));
+
+const props = withDefaults(defineProps<Props>(), {
   active: false,
 });
 </script>
 
 <template>
-  <LabelText>
-    <slot />
-  </LabelText>
+  <div :class="['pill', classes]">
+    <LabelText>
+      <slot />
+    </LabelText>
+  </div>
 </template>
+
+<style lang="scss" scoped>
+.pill {
+  border-color: rgba(var(--color-neutral-border), 1);
+  border-radius: var(--radius-medium);
+  border-style: solid;
+  border-width: var(--spacing-pico);
+  background-color: rgb(var(--color-neutral-bg));
+  color: rgb(var(--color-neutral-fg));
+  padding-inline: var(--spacing-small);
```

Comment:
`padding-inline` a `padding-block` jsou super css property, ale jak to už s novinkami bývá, tak nejsou plně podporovány. Obě property mají podporu jen v 91.41 % používaných zařízení. To je o 8.59 % míň, než "obyčejný" padding. To znamená, že asi každý 11. uživatel to bude mít rozbitý...

Narozdíl od `padding-block` a `padding-inline`, padding jde napsat v jednom řádku.

Can I use: [padding-block](https://caniuse.com/mdn-css_properties_padding-block) a [padding-inline](https://caniuse.com/?search=padding-inline).

Reply:
Ano, máš pravdu. Změnila jsem na `padding`.